markdown/preface.markdown

```diff
@@ -202,33 +210,38 @@ Another change is the way Python 3 treats integer division, which is the way mos
 We were also pretty excited about the new `@` *matrix multiplication* operator introduced in Python 3.5 in 2015.
 Check out chapters 5 and 6 for some examples of this operator in use!
 
-Possibly the biggest improvement in Python 3 is its support for Unicode, a way of encoding text that allows one to use not just the English alphabet, but rather any alphabet in the world.
+Possibly the biggest improvement in Python 3 is its support for Unicode, a way of encoding text that allows one to use not just the English alphabet, but any alphabet in the world.
 Python 2 allowed you to define a Unicode string, like so:
 
 ```python
 beta = u"β"
 ```
 
-But in Python 3, *everything* is unicode:
+But in Python 3, *everything* is Unicode:
 
 ```python
 β = 0.5
 print(2 * β)
 ```
 
 The Python core team decided, rightly, that it was worth supporting characters from all languages as first-class citizens in Python code.
-This is especially true now, when most new coders are not coming from English-speaking countries.
+This is especially true now, when most new coders are from non-English-speaking countries.
```

Comment:
I prefer the current wording. Both location and "first language" are only approximations to English ability/affinity. But I've had "first language" used as an impediment for visas more often. =P